docs/features.rst

```diff
@@ -56,18 +58,20 @@ replaced by hyphens. Names of positional arguments are used unmodified::
 
     optional arguments:
       -h, --help            show this help message and exit
-      --keyword-arg KEYWORD_ARG
+      -k KEYWORD_ARG, --keyword-arg KEYWORD_ARG
 
 In Python 3, any keyword-only arguments without defaults are marked as required.
 
-If you wish to specify that a particular flag should also have an associated
-short version, you can pass a mapping to `defopt.run`::
+You can add alternative short flags to arguments by passing a dictionary
+to `defopt.run` which maps flag names to single letters.  You can
+prevent automatic addition of short flags by passing an empty dictionary
+as argument::
```

Comment:
After rereading this section, I think it may need to be reworded - specifically "add alternative short flags" may be taken to imply that the given flags are used in addition to the generated ones, as opposed to overriding them entirely. How about something like this?

> You can override these generated short flags by passing a dictionary to `defopt.run` which maps flag names to single letters::
> 
> [example]
> 
> Passing an empty dictionary will suppress automatic short flag generation without adding new flags.

I think the second part belongs after the example is it's not actually demonstrated in there. How does that seem to you?